andela/ah-the-answer-frontend
# # 164829323 password reset

#### What does this PR do?
Configures the reset password page

#### Description of the task to be completed
- Render a password reset request form
- Validate user input 
- Render errors returned from the API

#### How should this be manually tested?
- Clone the repo and navigate to the root directory using a terminal
- Run the test suite by typing `npm test` or `npm test -- --coverage` to see the coverage

#### Relevant PT stories
https://www.pivotaltracker.com/story/show/164829323

#### Relevant Screenshots

<img width="822" alt="Screenshot 2019-05-10 at 04 57 07" src="https://user-images.githubusercontent.com/22986772/57508713-ecba2480-730a-11e9-8e5a-d121d4817633.png">
<img width="839" alt="Screenshot 2019-05-10 at 10 02 05" src="https://user-images.githubusercontent.com/22986772/57508719-f3489c00-730a-11e9-92d3-9149d00da610.png">
<img width="745" alt="Screenshot 2019-05-10 at 04 57 37" src="https://user-images.githubusercontent.com/22986772/57508694-e330bc80-730a-11e9-966b-20394cd5ea55.png">
<img width="612" alt="Screenshot 2019-05-10 at 10 07 37" src="https://user-images.githubusercontent.com/22986772/57508936-93062a00-730b-11e9-8def-20d337ec4a7a.png">
